starter-code/pizza.js

```diff
@@ -27,59 +42,166 @@ function renderEverything() {
   renderGreenPeppers()
   renderWhiteSauce()
   renderGlutenFreeCrust()
-
   renderButtons()
   renderPrice()
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll('.pep').forEach(function ($pep) {
     if (state.pepperonni) {
       $pep.style.visibility = "visible";
-    }
-    else {
+    } else {
       $pep.style.visibility = "hidden";
     }
   })
 }
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(function ($mush) {
```

Comment:
Good exercise! Did u notice that you repeated the same logic through the file? 
1) querySelectorAll(something)
2) If something is true, style.visibility = "visible"; if not style.visibility = "hidden";

Next time, if you are feeling that you are repeating the same logic try to create a function to encapsulate that logic.  🚀

Reply:
Good point and will do!